projects/xmlpull/build.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash -eu
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+mv $SRC/{*.zip,*.dict} $OUT
+
+cat > patch.diff <<- EOM
+diff --git a/pom.xml b/pom.xml
+index 3e29db9..c79e086 100644
+--- a/pom.xml	2022-04-14 17:14:14.830692400 +0200
++++ b/pom.xml	2022-04-14 17:33:25.535451222 +0200
```

Comment:
This looks a bit brittle. What's the reason why we need to remove this?

Reply:
Agreed. I temporarily removed the javadoc because of dependency issue with maven. I'll look into fixing that.